mevislab.github.io/content/tutorials/visualization/pathtracer/pathtracerexample2.md

```diff
@@ -0,0 +1,114 @@
+
+## Steps to do
+### Develop your network
+
+Begin by loading the example [medical images](/examples/visualization/example6/Volume_1.mlimage) using the `LocalImage` module. Connect it to a `View2D` to visually inspect its contents.
+
+![MR Image of Knee](/images/tutorials/visualization/pathtracer/V6.2_1.png "MR Image of Knee")
+
+
+Connect your `LocalImage` to the `SoPathTracerVolume` module to initiate volume rendering. Add the `SoPathTracer` module into your SDK and connect both to the `SoExaminerViewer` module to visualize the rendered volume. the `SoPathTracer` module serves as the main rendering component. It offers parameters for resulotion, camera settings, background colors and iteration control.
+
+{{<alert class="info" caption="Additional Info">}}
+It's essential to consistently position the `SoPathTracer `module on the right side of the scene. This strategic placement ensures that the module can render all objects located in the scene before it accurately. 
+{{</alert>}}
+
+![SoPathTracerVolume & SoPathTracer](/images/tutorials/visualization/pathtracer/V6.2_2.png "SoPathTracerVolume & SoPathTracer")
+
+If you check your `SoExaminerViewer` you will see a black box, this indicates often a need for additional adjustments to color, light and transparency.
+
+![SoExaminerViewer](/images/tutorials/visualization/pathtracer/V6.2_3.png "SoExaminerViewer")
+
+To brighten up your scene, connect both the `SoPathTracerAreaLight` and `SoPathTracerBackgroundLight` modules to your `SoExaminerViewer` to improve scene lighting. 
+
+The `SoPathTracerAreaLight` shapes and colors specific areas realistically, while `SoPathTracerBackgroundLight` illuminates the whole scene with your chosen three colors. These lights make your scene vibrant, providing a solid base for further adjustments with other modules.
+
+![Lights](/images/tutorials/visualization/pathtracer/V6.2_4.png "Lights")
+
+Connect `SoLUTEditor` to your `SoPathTracerVolume` to visualize the knee. However, it might initially appear in black and with shading.
+
+![SoLUTEditor](/images/tutorials/visualization/pathtracer/V6.2_5_SoLUTEditor.png "SoLUTEditor")
+
+
+To easily identify your minimum and maximum range, connect the `MinMaxScan` module to your `LocalImage`. Open it's panel and read your *Static Min Value* and *Static Max Value*. This step allows you to set the appropriate values with convenience.
+
+![MinMaxScan](/images/tutorials/visualization/pathtracer/V6_MinMaxScan.png "MinMaxScan")
+
+Then open the panel of the `SoLUTEditor` and select tab *Range*. Define *New Range Min* as *0* and *New Range Max* as *2047*. Once done, manually define your desired colors for your LUT in the *Editor* tab.
+
+{{<alert class="info" caption="Additional Info">}}
+You can replace `SoLUTEditor` with `LUTLoad` and load the attached xml files to store the same adjustments.
```

Comment:
which attached file? Provide a link here and let them download the LUT here directly.